### PR TITLE
[PLS] Dark Suspicions

### PR DESCRIPTION
**Pull request by Filipecordeiro**

### CODE REVIEW

File: Mage.Sets/src/mage/cards/d/DarkSuspicions.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+    @Override
+    public mage.cards.d.DarkSuspicionsEffect copy() {
+        return new mage.cards.d.DarkSuspicionsEffect(this);
+    }
+
+    @Override
+    public boolean apply(Game game, Ability source) {
+        Player controller = game.getPlayer(source.getControllerId());
+        Player opponent = game.getPlayer(this.getTargetPointer().getFirst(game, source));
```
Comment (LevelX2) [inline Mage.Sets/src/mage/cards/d/DarkSuspicions.java]: add a check that the controller != null

File: Mage.Sets/src/mage/cards/d/DarkSuspicions.java [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+        return new DarkSuspicions(this);
+    }
+}
+
+class DarkSuspicionsEffect extends OneShotEffect {
+
+    public DarkSuspicionsEffect() {
+        super(Outcome.LoseLife);
+        staticText = "that player loses X life, where X is the number of cards in that player's hand minus the number of cards in your hand.";
+    }
```
Comment (LevelX2) [inline Mage.Sets/src/mage/cards/d/DarkSuspicions.java]: no need for dots at the end of the effect text, it's added automatically

Comment (Filipecordeiro) [inline Mage.Sets/src/mage/cards/d/DarkSuspicions.java]: Dot removed

**Comment by LevelX2** [comment]
Thanks for your contribution.

**Pull request merged**